pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/QuickstartRunner.java

```diff
@@ -132,6 +136,7 @@ private void startBrokers()
     for (int i = 0; i < _numBrokers; i++) {
       StartBrokerCommand brokerStarter = new StartBrokerCommand();
       brokerStarter.setPort(DEFAULT_BROKER_PORT + i)
+          .setBrokerMultiStageRunnerPort(DEFAULT_BROKER_MULTISTAGE_RUNNER_PORT - i)
```

Comment:
Can we use `+` instead of `-` since it's more intuitive? I know you did it because the `DEFAULT_SERVER_MULTISTAGE_RUNNER_PORT` is 8422. But can we update the value of `DEFAULT_SERVER_MULTISTAGE_RUNNER_PORT` to sth like 8442?

Reply:
sounds good